### DISCUSSION TITLE
Tethered, 3 phase 32A, PP wire?

### DISCUSSION THREAD

**Discussion by nagyrobi** [dzurikmiroslav/esp32-evse#55] [General]
I'd like to build the unit as tethered. Without a socket, just a fixed installed cable. Questions:
- I see that available cables for this rating are 5G6+1x0,75. Wonder why is the second 0,75 wire missing. How does the car signal back without the PP wire? What am I missing?
- If I omit the socket, is it OK to also omit from soldering on the board the components from the "LOCK" schematic?
- Is it enough to disable in software to set via ModBUS register 300 to 0, to disable the socket? Will it be persistent setting?

**Comment by dzurikmiroslav** [comment]
PP pin tells about the maximum current that the cable can handle, is installed in Type2 plugs on each side.
Yes you can omit "LOCK" parts in schematics, you should modify board config to SOCKET_LOCK=n, so that the controller does not report an error if the Socket Outlet is turned on.
Socket Outlet settings (register 300 or WebUI) is persistent.

**Reply by nagyrobi** [reply]
Socket Outlet settings (register 300 or WebUI) is equivalent to setting SOCKET_LOCK=n?

**Reply by dzurikmiroslav** [reply]
No, config SOCKET_LOCK=n tells there is no socket lock in schematics.
Socket Outlet settings (register 300 or WebUI) tells if should detect PP max current before charging and handle socket lock if is available.

**Reply by nagyrobi** [reply]
Thank you.
